scripts/README.md

```diff
@@ -53,38 +53,27 @@ as it build NLCD tiles only where corresponding NED tile are available.
 So this process shall be done after a snapshot of the NED terrain tile is built.
       
 
-## Process for recreating census tract GeoJSON
-
-There are three steps to recreate the census tract GeoJSON database as follows:
-
-  1. Download all original census tract data from the USGS web site. 
-       `$ python extract_census_tracts_json.py --retrieve`
-    `Spectrum-Access-System/data/census` will be created and all original census tract 
-    shapefile will be downloaded into this directory. Note that the current GitHub 
-    repository provide that directory with the original files from 2010 Census, so this
-    step is unnecessary.
-
-  2. Convert the shape files to GeoJSON.
-       `$ python extract_census_tracts_json.py --convert`
-    The shapefile from directory `Spectrum-Access-System/data/census` will be converted
-    to GeoJSON file within the same directory.
-
-  3. Split the GeoJSON file into individual census tract files based on GEOID.
-       `$ python extract_census_tracts_json.py --split`
-    The converted census tract files will be split in individual GeoJSON tract file
-    and dumped into the `Spectrum-Access-System/data/census_tracts` directory.
+## Process for recreating counties GeoJSON
+we have two options to get the county data:
```

Comment:
"We have ..."
"The resulting json ..."
Also let an empty line so the markdown formatting is right for 1) and 2) items.

Please verify that the formatting done properly by looking at the final page in GitHub ("view file").